examples/todo/imports/ui/components/uis.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Tunnel } from 'react-tunnels';
+
+export const updateAppTitle = title => (
+  <Tunnel id="app-title">{title || 'Future Web'}</Tunnel>
```

Comment:
Why don't you update the window.title too? It's good for browser history and Google Analytics ;)

Reply:
Ahhhh haha,  I didn't know that